docs/notifier-catalog.md

```diff
@@ -1746,10 +1738,151 @@ Requires:
 
 ### `ntfy`
 
-Support for [ntfy] is provided through Apprise, see [apprise_single](#apprise_single)
-and [apprise_multi](#apprise_multi).
+> [ntfy] (pronounce: _notify_) is a simple HTTP-based [pub-sub] notification service.
+> It allows you to send notifications to your phone or desktop via scripts from
+> any computer, entirely without signup, cost or setup.
+> [ntfy is also open source](https://github.com/binwiederhier/ntfy), if you want to
+> run an instance on your own premises.
+
+ntfy uses topics to address communication channels. This topic is part of the
+HTTP API URL.
+
+To use the hosted variant on `ntfy.sh`, just provide an URL including the topic.
+```ini
+[config:ntfy]
+targets  = {
+    'test': 'https://ntfy.sh/testdrive',
+    }
+```
+
+When running your own instance, you would use a custom URL here.
+```ini
+[config:ntfy]
+targets  = {
+    'test': 'http://username:password@localhost:5555/testdrive',
+    }
+```
+
+In order to specify more options, please wrap your ntfy URL into a dictionary
+under the `url` key. This way, additional options can be added.
+```ini
+[config:ntfy]
+targets  = {
+    'test': {
+        'url': 'https://ntfy.sh/testdrive',
+        },
+    }
+```
+
+:::{important}
+[ntfy publishing options] outlines different ways to marshal data to the ntfy
+HTTP API. mqttwarn is using HTTP headers for serializing values, because the
+HTTP body will already be used for the attachment file. Because of this, you
+are not able to use UTF-8 characters within your message text, they will be
+replaced by placeholder characters like `?`.
+:::
```

Comment:
As I said in the other comment; ntfy server does support UTF-8 in headers, so if you send valid UTF-8 in headers it'll work. But because the HTTP spec does not officially support UTF-8, your libraries or language may not support it. 

The suggested RFC encoding would fix this, obviously.

Reply:
Hi Philipp.

> ntfy server does support UTF-8 in headers, so if you send valid UTF-8 in headers it'll work.

Ah. Thanks for mentioning it once again.

> But because the HTTP spec does not officially support UTF-8, your libraries or language may not support it.

Right, the Python client libraries would currently already croak on that detail, raising a corresponding exception. I will see if a workaround can be applied in form of a monkey patch or such.

But we agree it's dangerous, because there may be HTTP intermediaries in between, which might scramble those headers, right?

> Encoding the headers using RFC 2047 would fix this, obviously.

I will very much appreciate such an improvement to ntfy. Until it will converge, I will eventually look into the monkey patch solution.

Thank you so much for your swift responses, and keep up the spirit.

With kind regards,
Andreas.

Reply:
Works wonderfully: https://github.com/binwiederhier/ntfy/pull/707

Reply:
It's merged and will be in the next release

Reply:
That was fast. Thank you very much!

Reply:
I've adjusted the client-side implementation correspondingly with 65c4c11ff5. Thank you again!

Reply:
Hi again,

at https://github.com/jpmens/mqttwarn/commit/609d79f9400850989b5435570126f8bd70fa6be0#r110220447, you've commented:

> I didn't do tags. Only title and message. Though tags are probably a good idea too.

This has already been added with https://github.com/binwiederhier/ntfy/commit/59a5077713 now - thank you so much! I've complemented this improvement with b67e39077ede, where RFC 2047 encoding will now only be applied to specific fields, `title`, `message`, and `tags`.

When looking through the available fields once more, I only discovered that the Action's `label` value might be another candidate for being encoded with RFC 2047, in order to use UTF-8 characters on the action buttons, when using HTTP header transport mode. But please consider this only as a discovery on my end, I personally do not have a need for that.

Thanks again for the quick turnaround on this matter.

With kind regards,
Andreas.